src/proxy-ui-api/frontend/src/views/AddSubsystem/AddSubsystem.vue

```diff
@@ -1,83 +1,317 @@
+
 <template>
-  <v-container class="xrd-tab-max-width">
-    <v-layout text-xs-center wrap column>
-      <v-flex mb-4>
-        <h1 class="display-1 font-weight-bold mb-3">Add a Subsystem</h1>
-        <p
-          class="subheading font-weight-regular"
-        >Specify the name of the Subsystem and add it to the selected Member.</p>
-      </v-flex>
-      <v-layout pa-4 column>
-        <v-layout mb-4 row justify-start>
-          <v-flex mb-4 class="input-title">Member Name</v-flex>
-          <v-flex mb-4 class="subs-input">{{member.name}}</v-flex>
-        </v-layout>
-        <v-layout mb-4 row justify-start>
-          <v-flex mb-4 class="input-title">Member Class</v-flex>
-          <v-flex mb-4 class="subs-input">{{member.class}}</v-flex>
-        </v-layout>
-        <v-layout mb-4 row justify-start>
-          <v-flex mb-4 class="input-title">Member Code</v-flex>
-          <v-flex mb-4 class="subs-input">{{member.code}}</v-flex>
-        </v-layout>
-
-        <v-layout mb-4 row justify-start>
-          <v-flex mb-4 class="input-title">Subsystem Name</v-flex>
-          <v-text-field label="Regular" class="input-name"></v-text-field>
-        </v-layout>
-      </v-layout>
-
-      <v-layout mb-4 justify-space-between>
-        <v-btn color="primary" @click="cancel" rounded outlined class="mb-2 rounded-button">Cancel</v-btn>
-
-        <v-btn
-          color="primary"
-          @click="addSubsystem"
-          rounded
-          dark
-          class="mb-2 rounded-button"
-        >Add subsystem</v-btn>
-      </v-layout>
-    </v-layout>
-  </v-container>
+  <div class="view-wrap">
+    <subViewTitle class="view-title" :title="$t('wizard.subsystem.title')" :showClose="false" />
+
+    <div class="content">
+      <div class="info-block">
+        <div>
+          {{$t('wizard.subsystem.info1')}}
+          <br />
+          <br />
+          {{$t('wizard.subsystem.info2')}}
+        </div>
+        <div class="action-block">
+          <large-button
+            @click="showSelectClient = true"
+            outlined
+            data-test="select-subsystem-button"
+          >{{$t('wizard.subsystem.selectSubsystem')}}</large-button>
+        </div>
+      </div>
+
+      <ValidationObserver ref="form2" v-slot="{ validate, invalid }">
+        <div class="row-wrap">
+          <FormLabel labelText="wizard.memberName" helpText="wizard.client.memberNameTooltip" />
+          <div v-if="client" data-test="selected-member-name">{{client.member_name}}</div>
```

Comment:
at least 3 data-test -attributes with the 'selected-member-name' -value

Reply:
Fixed